### DISCUSSION TITLE
Using Cloudinary upload plugin it returns url without extension

### DISCUSSION THREAD

**Discussion by seleckis** [strapi/strapi#6647] [Legacy Topics]
I want to use SVG image for background on my website. I use Cloudinary  as a provider for media library. After uploading, setting media in content, and using GraphQL query:
```
{
  pages {
    bg {
      url      
    }
  }
}
```
I get full raw url of that file like this:
`https://res.cloudinary.com/dyry2to5c/raw/upload/v1591948235/media_Design_bg_76fe252a25`

It is returned as raw file without extension. When opening this url in browser the file is downloaded. But it is not working as a background in css.

So the question is, how to get an image with extension?

**Comment by RaphaelFreire** [comment]
When uploading svg there was also the problem of not uploading the image thumbnail.

https://react-api-raphaelfreire.herokuapp.com/landing-page

**Comment by derrickmehaffy** [comment]
Hi @RaphaelFreire we are currently looking into this internally and once we have an answer we will let you know.

**Comment by lauriejim** [comment]
Hello! I just tried myself and everything is working fine on my side.
Here is a video record of my test - https://www.loom.com/share/da8fc1d7aa54407cbcedf823f3442b81
Let me know if I'm missing something

**Reply by seleckis** [reply]
Hi, thanks for testing. Yes, JPEG works, but not SVG

**Reply by RaphaelFreire** [reply]
Hi, @seleckis I add `<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 130 128">` in my svg and works.

**Reply by seleckis** [reply]
And fetching it with graphql you receive url with extension `.svg`?

**Reply by RaphaelFreire** [reply]
Yes!

https://react-api-raphaelfreire.herokuapp.com/landing-page
"url": "https://res.cloudinary.com/drgnqntag/image/upload/v1598491850/react_791425b954_76e5e45bc0.svg",

**Comment by paulajbastos** [comment]
I have the same problem as @seleckis!

no .svg extension

**File in strapi**
![Screen Shot 2020-10-01 at 19 23 51](https://user-images.githubusercontent.com/374666/94869472-c626ed00-041b-11eb-8255-055bc15addb1.png)

**File in Cloudinary**
![Screen Shot 2020-10-01 at 19 24 09](https://user-images.githubusercontent.com/374666/94869494-d0e18200-041b-11eb-9a8c-6e2e5c90ca40.png)

**Reply by paulajbastos** [reply]
Found the solution!
Need to put `<?xml version="1.0" encoding="utf-8"?>` before the `<svg>` tag.

**Reply by carmelodiventi** [reply]
Thanks

**Reply by bahadryalcn** [reply]
Where should we add this line exactly?

**Reply by strazan** [reply]
> Where should we add this line exactly?

At the top of the file, above the `<svg>` tag!

**Comment by derrickmehaffy** [comment]
Locking as migrated to: https://forum.strapi.io/t/using-cloudinary-upload-plugin-it-returns-url-without-extension-6647/1538
